commands/wait.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function * run (context, heroku) {
+  const fetcher = require('../lib/fetcher')
+  const host = require('../lib/host')
+  const app = context.app
+  const db = context.args.database
+
+  let waitFor = co.wrap(function * waitFor (db) {
+    const wait = require('co-wait')
+    let interval = parseInt(context.flags['wait-interval'])
+    if (!interval || interval < 0) interval = 1
```

Comment:
Is this what we default to now? That seems rather low, considering that the average wait is probably a couple of minutes at least (for fresh databases) or up to hours (for forks and followers).